app/views/users/index.html.erb

```diff
@@ -1,13 +1,11 @@
 <h1>Users</h1>
 <ul>
   <% @users.each do |user| %>
-    <li>
-      <a href="<%= user_posts_path(user) %>">
-        <div class="user-box">
-          <%= render partial: 'user_helper/user_photo', locals: { user: user } %>
-          <%= render partial: 'user_helper/user_details', locals: { user: user } %>
-        </div>
-      </a>
-    </li>
+    <a href="<%= user_posts_path(user) %>">
+      <div class="user-box">
+        <%= render partial: 'shared/user_photo', locals: { user: user } %>
+        <%= render partial: 'shared/user_details', locals: { user: user } %>
+      </div>
+    </a>
```

Comment:
- [ ] [OPTIONAL] kindly fix the way the user image looks in your design
![Screenshot from 2023-09-09 15-35-08](https://github.com/otmaneechchafyky/blog-app/assets/66097371/2c3c34c5-26d7-48cf-9eab-f41bd543057d)